spec/rubocop/cop/rails/reversible_migration_spec.rb

```diff
@@ -121,6 +121,10 @@ def change
     it_behaves_like 'offense', 'drop_table(without block)', <<~RUBY
       drop_table :users
     RUBY
+
+    it_behaves_like 'offense', 'drop_table(without block)', <<~RUBY
```

Comment:
It seems that it should result in no offense as it is still in a state where judgment cannot be made. Also, the description string for the spec appear to have been left as they were copied and pasted.

```suggestion
    it_behaves_like 'accepts', 'drop_table (without arguments)', <<~RUBY
```

Reply:
I switched it over to no offense, that seems fine. And yeah, failed to change the example description.